app/res/values/dimens.xml

```diff
@@ -142,4 +142,5 @@
     <dimen name="map_info_view_text_padding">4dp</dimen>
     
     <dimen name="dp40">40dp</dimen>
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/5a959df2d1681e0a640ebb72d4513eaecefa585e/app/res/values/dimens.xml#L144"></span>The resource <code>R.dimen.dp40</code> appears to be unused</td>
    </tr>
  </tbody>
</table>